sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ManifestFileCommitProtocol.scala

```diff
@@ -43,6 +43,9 @@ class ManifestFileCommitProtocol(jobId: String, path: String)
   @transient private var fileLog: FileStreamSinkLog = _
   private var batchId: Long = _
 
+  /** returns copied version of addedFiles: only unit tests purpose */
+  private[sql] def getAddedFiles: Array[String] = Array(addedFiles: _*)
```

Comment:
this method is not needed once you remove TaskFailToCommitManifestFileCommitProtocol